lib/config/config-cache.js

```diff
@@ -0,0 +1,150 @@
+/**
```

Comment:
We now have two config caches. One for config files themselves (used in file-finder.js) and one for config vectors. Do we need both of them?